### DISCUSSION TITLE
【FAQ】WTF Academy习题系统常见问题汇总

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#260] [General]
### 1. 我提交了作业，但看不到我的成绩。

如果你遇到了这个问题，可能有以下几点原因：
a. 你没登陆，需要点网站右上角，利用github登陆 https://wtf.academy
b. 你提交作业时，填的邮箱不是github的主邮箱（primary github）。你可以在你的github邮箱页面查询你的primary email： https://github.com/settings/emails
c. 你的网络连接有问题。

### 2. 我重新提交了作业，但成绩没有刷新。

成绩会记录你的历史最高成绩，如果你重新提交作业但成绩没有上升，则不会刷新成绩。

### 3. 我重新提交了作业，有错题，但没有显示错误选项/原因。

我们特意隐藏了错误选项/原因，目的是让学员通过阅读教程，查询资料，自己发现错误的题目和原因。题目均为单选题，数量较少，因此若告知错误选项/原因，将大幅降低习题难度，达不到学习目的。并且这也符合智能合约的特点：安全第一。如果学员基础掌握的不好，写智能合约出漏洞时，黑客不会告诉你哪里写错了，而会直接盗走用户资产。

**Comment by YCChan2001** [comment]
试了n遍，还是没有成绩。。。

**Reply by LBJyang** [reply]
那您老人家还弄个查看准确率干嘛...

**Comment by LBJyang** [comment]
BOSS，好像真的不对哎，答题的答案不论怎么答，都不更新成绩啊

**Reply by AmazingAng** [reply]
多试几个github邮箱

**Comment by zhy854001978** [comment]
我提交了测试，有错误，但是不知道错误原因

**Reply by AmazingAng** [reply]
不显示错误原因，要自己找

**Comment by EthanOK** [comment]
至少应该显示哪道题错了吧

**Comment by SadPencil** [comment]
登录，不是陆，注意错别字

**Comment by YCChan2001** [comment]
关于： 我提交了作业，但看不到我的成绩。
补充一个点吧，填写那个谷歌表单邮箱的时候，要手打一遍。之前一直没有成绩就是因为这个问题。

**Reply by 637leaf** [reply]
thanks

**Comment by hanwudi9527** [comment]
我上午还正常，提交之后刷新就可以看到自己的成绩了，但是下午就不行了，我确认已经登录，并且电子邮件是github的主邮箱，这是什么原因呢

**Reply by AmazingAng** [reply]
有没有可能是网络问题

**Reply by hanwudi9527** [reply]
不应该吧，我刷新的话，可以刷新，网络没什么问题

**Reply by hanwudi9527** [reply]
@AmazingAng 我换了个github邮箱，答题提交后成绩任然是0，可以帮我看下吗，非常感谢

**Comment by chenso121** [comment]
solitidy应用部分能不能也出题，感觉做题能发现很多没注意的细节

**Comment by SolZm** [comment]
我提交了作业，但看不到我的成绩。
 前两个方法都没有问题，登录，邮箱都正确，但是还是不显示地址
 第三个网路问题，是代理地址必须是某个节点还是都可以，
 试了好多次了，还是不行。
 其他的小伙伴，有解决的吗？

**Reply by SolZm** [reply]
解决了，把github邮箱Primary换成  qq邮箱可以了····

**Comment by lingchingteng** [comment]
可以弄一個測試用的測驗讓我試試看到底是哪裡沒串好導致都沒有分數嗎？
第一課就過不了關，又找不到答案QQ

**Reply by AmazingAng** [reply]
不可以对答案

**Comment by Kungem007** [comment]
第15讲,教程里面写的是“require唯一的缺点就是gas随着描述异常的字符串长度增加”一个影响因素,做的题里面选的是‘require消耗gas的数量会随着检查条件的增多以及"异常的描述"变长而增加。”’两个影响因素,建议订正一下

**Reply by AmazingAng** [reply]
嗯，前面说的是缺点（1个），测试考的是特性（2个），不冲突，需要学员对所学知识有一定的拓展。

**Comment by pacinoxiey** [comment]
我更换不同答案，得分竟然是一样的

**Reply by keyzhao0118** [reply]
得分只会记录你的最高分，也就是说你更换答案后如果分数更低的话，是不会刷新你的分数的

**Comment by brefontaire** [comment]
Quiz_30_TryCatch 第二题
<img width="656" alt="Screenshot 2023-06-29 at 14 18 24" src="https://github.com/AmazingAng/WTF-Solidity/assets/136898301/a8fda690-60a5-4447-8726-7efa867d5650">
正确答案应该是 **assert()** 吧，但选了 **以上都是** 才过。

**Reply by Johnny0song** [reply]
谢谢

**Comment by brefontaire** [comment]
Quiz_23_ Delegatecall 第五题
<img width="660" alt="Screenshot 2023-06-29 at 16 01 13" src="https://github.com/AmazingAng/WTF-Solidity/assets/136898301/1da91027-3a3e-418f-9290-9b08d84cb2a4">
所有选项的括号都不匹配，结尾少了一个 **)**

**Reply by Johnny0song** [reply]
赞 看得真仔细

**Comment by esters18443** [comment]
最好做完能看到当前分数，而不是只显示最好成绩

**Reply by liquidjsh** [reply]
赞同

**Comment by CCypher123** [comment]
我反复尝试，然而还是没有成绩。
我确认已经登录，并输入的是github的primary邮箱，网络也无问题（无论是跳转网页还是提交表单）
为此，我还更改了三次primary邮箱，outlook/gmail/qq都尝试过了，提交完全正常但仍然没有成绩。
我不知道哪里出了问题。

**Reply by AmazingAng** [reply]
你在网站点右上角个人资料，看一下邮箱是否和你填的一致

**Reply by CCypher123** [reply]
您好，如果是wtf网站的个人资料，我没有看到里面有显示我的邮箱。
如果是github的个人资料，我在此前已经反复确认过了。

**Reply by HouttuyniaVibe** [reply]
我也遇到了同样的问题
